CHANGELOG.md

```diff
@@ -43,12 +80,13 @@ Features listed in the internal section are not ready to use but, in the spirit
 
 ### Features:
 - The GDK now uses SpatialOS `14.1.0`.
+- Added in-editor support for exposing a local runtime at a particular IP address. This offers the same functionality as the `--runtime_ip` option in the SpatialOS CLI.
```

Comment:
Hmmmm, these seem in the wrong place actually, they shouldn't be going into 0.7.0-preview should they?